internal/server/metrics/metrics.go

```diff
@@ -55,12 +55,12 @@ var (
 
 	// Attributes used in evaluation metrics
 	//nolint
-	AttributeMatch       = attribute.Key("flipt.match")
-	AttributeFlag        = attribute.Key("flipt.flag")
-	AttributeFlagType    = attribute.Key("flipt.flag_type")
-	AttributeSegments    = attribute.Key("flipt.segments")
-	AttributeReason      = attribute.Key("flipt.reason")
-	AttributeValue       = attribute.Key("flipt.value")
-	AttributeNamespace   = attribute.Key("flipt.namespace")
-	AttributeEnvironment = attribute.Key("flipt.environment")
+	AttributeMatch       = attribute.Key("flipt_match")
+	AttributeFlag        = attribute.Key("flipt_flag")
+	AttributeFlagType    = attribute.Key("flipt_flag_type")
+	AttributeSegments    = attribute.Key("flipt_segments")
+	AttributeReason      = attribute.Key("flipt_reason")
+	AttributeValue       = attribute.Key("flipt_value")
+	AttributeNamespace   = attribute.Key("flipt_namespace")
+	AttributeEnvironment = attribute.Key("flipt_environment")
```

Comment:
As fyi. I am okay with this. But prometheus and OTLP have conflict here. Per opentelemetry guidelines `dot` is recommended. 

> For example, use container.csi.* instead of container.container_storage_interface or container.oci.* instead of container.open_container_initiative.*

https://opentelemetry.io/docs/specs/semconv/general/naming/

Reply:
we could use `.` for tracing attributes and `_` for metrics? its tough because prometheus recommends or converts `.` to `_` which was messing me up in the queries because i was querying for `flipt.environment` but prometheus was storing it as `flipt_environment`